### PR TITLE
Config Media - blue bar with Fieldset Description

### PR DESCRIPTION
**Pull request by pe7er**
Most **Component Configuration Options** have under the option tab **a blue bar with a description**

### Summary of Changes

This PR **changes the tab option name** from [Component] to [Upload Settings] and **adds a blue bar** with description.

#### Before the PR:
backend: System > Global Configuration > (in Left menu) Media > [**Component**]

info-label-options-media-component-before

#### After the PR:
backend: System > Global Configuration > (in Left menu) Media > [**Upload Settings**]

info-label-options-media-component-after

### CODE REVIEW

**Comment by pe7er** [comment]
I do not understand why Travis is unhappy. 
If someone can shed a light on that then I might be able to improve this PR. 
Thanks!

**Comment by zero-24** [comment]
@pe7er see: https://github.com/joomla/joomla-cms/pull/12096

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 05e7017ef7b1cad616f3c8ee41ede3f17d8dc227<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12088">issues.joomla.org/joomla-cms/12088</a>.</sub>

**Comment by pe7er** [comment]
@zero-24 Thanks for that explanation & for testing!

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 05e7017ef7b1cad616f3c8ee41ede3f17d8dc227<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12088">issues.joomla.org/joomla-cms/12088</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 05e7017ef7b1cad616f3c8ee41ede3f17d8dc227<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12088">issues.joomla.org/joomla-cms/12088</a>.</sub>

File: administrator/language/en-GB/en-GB.com_media.ini [hunk 1]
```diff
@@ -12,6 +12,8 @@ COM_MEDIA_CAPTION_CLASS_LABEL="Caption Class"
 COM_MEDIA_CAPTION_CLASS_DESC="This will apply the entered class to the '<figcaption>' element. For example: 'text-left', 'text-right', 'text-center'."
 COM_MEDIA_CLEAR_LIST="Clear List"
 COM_MEDIA_CONFIGURATION="Media: Options"
+COM_MEDIA_CONFIG_UPLOAD_LABEL="Upload Settings"
+COM_MEDIA_CONFIG_UPLOAD_FIELDSET_DESC="Configure the media upload settings to increase security."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_media.ini]: This is not correct - you could use these options to DECREASE the security settings

**Comment by brianteeman** [comment]
This is not correct - see comments inline

**Comment by pe7er** [comment]
@brianteeman What would be correct? "Configure the media upload settings."

**Comment by brianteeman** [comment]
The only correct string to cover all the current options and the potential future options would be to say these are the options for the media manager component. But the blue bar is not needed -this would just add meaningless text in order to justify adding th blue bar - my 2c

**Comment by pe7er** [comment]
Ok. I understand your point but if every **Options tab starts with a blue bar with some information**, **the UI for Options** will be **consistently the same on all Option Tabs**. That way an admin has an extra visual pointer (besides the "Options" in the Page Title) that they are on the Options page.

**Comment by brianteeman** [comment]
I don't agree in adding help text if it has no meaning or value. That has
the effect of users just ignoring it. It should only exist if it is
required information

**Comment by korneliusz401** [comment]
It has been tested succesfully ✅.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12088">issues.joomla.org/tracker/joomla-cms/12088</a>.</sub>

**Comment by Quy** [comment]
I concur with Brian. The left sidebar menu is enough visual cue that I’m on the Options page.

**Comment by brianteeman** [comment]
Closed for the reasons stated above and elsewhere

**Pull request closed without merging**